app/views/provider/admin/bot_protections/_form.html.slim

```diff
@@ -0,0 +1,11 @@
+= semantic_form_for settings, url: url, html: {:id => 'bot-protection-settings' } do |form|
+  = form.inputs 'Protection against bots' do
```

Comment:
With this refactor we don't keep the "admin portal" and "developer portal" label in this title. Or it was considered redundant?

Actually, we probably can move this and the commit button labels to I18n, and reuse the complete view, not only the form.

Reply:
> With this refactor we don't keep the "admin portal" and "developer portal" label in this title. Or it was considered redundant?

I don't remember 🤕

> Actually, we probably can move this and the commit button labels to I18n, and reuse the complete view, not only the form.

I don't like the idea of reuse the complete view, because then we should set some parameters like `url` or `field` in the controller.

Reply:
OK, but we can still use I18n for `Protection against bots` and set it to different values for admin and dev portal.

Reply:
Here you go: https://github.com/3scale/porta/pull/3841/commits/1f9ef723d6f2edf877fe0e194fa43327cbe3f016